base/gfspapp/task_options.go

```diff
@@ -52,9 +52,9 @@ const (
 	// MaxMigratePieceTime defines the max timeout to migrate piece.
 	MaxMigratePieceTime int64 = 50
 	// MinMigrateGVGTime defines the min timeout to migrate gvg.
-	MinMigrateGVGTime int64 = 10
+	MinMigrateGVGTime int64 = 1800 // 0.5 hour
 	// MaxMigrateGVGTime defines the max timeout to migrate gvg.
-	MaxMigrateGVGTime int64 = 50
+	MaxMigrateGVGTime int64 = 3600 // 1 hour
```

Comment:
Is the 1 hour timeout also for GVGs with large data storage?

Reply:
if the task is running, the update time will be updated, and will not be judged as timed out ( update_time + this var > now).

Reply:
In other words, will the start time of the timeout be updated?

Reply:
> if the task is running, the update time will be updated, and will not be judged as timed out ( update_time + this var > now).

Okay, make sense.